aggregator/options.go

```diff
@@ -636,42 +735,89 @@ func (o *options) computeQuantiles() {
 	o.timerQuantiles, _ = o.DefaultTimerAggregationTypes().PooledQuantiles(o.QuantilesPool())
 }
 
-func (o *options) computeSuffixes() {
-	o.suffixes = make([][]byte, policy.MaxAggregationTypeID+1)
+func (o *options) computeDefaultSuffixes() {
+	o.defaultSuffixes = make([][]byte, policy.MaxAggregationTypeID+1)
 
 	for aggType := range policy.ValidAggregationTypes {
 		switch aggType {
 		case policy.Last:
-			o.suffixes[aggType.ID()] = o.AggregationLastSuffix()
-		case policy.Lower:
-			o.suffixes[aggType.ID()] = o.AggregationLowerSuffix()
-		case policy.Upper:
-			o.suffixes[aggType.ID()] = o.AggregationUpperSuffix()
+			o.defaultSuffixes[aggType.ID()] = o.AggregationLastSuffix()
+		case policy.Min:
+			o.defaultSuffixes[aggType.ID()] = o.AggregationMinSuffix()
+		case policy.Max:
+			o.defaultSuffixes[aggType.ID()] = o.AggregationMaxSuffix()
 		case policy.Mean:
-			o.suffixes[aggType.ID()] = o.AggregationMeanSuffix()
+			o.defaultSuffixes[aggType.ID()] = o.AggregationMeanSuffix()
 		case policy.Median:
-			o.suffixes[aggType.ID()] = o.AggregationMedianSuffix()
+			o.defaultSuffixes[aggType.ID()] = o.AggregationMedianSuffix()
 		case policy.Count:
-			o.suffixes[aggType.ID()] = o.AggregationCountSuffix()
+			o.defaultSuffixes[aggType.ID()] = o.AggregationCountSuffix()
 		case policy.Sum:
-			o.suffixes[aggType.ID()] = o.AggregationSumSuffix()
+			o.defaultSuffixes[aggType.ID()] = o.AggregationSumSuffix()
 		case policy.SumSq:
-			o.suffixes[aggType.ID()] = o.AggregationSumSqSuffix()
+			o.defaultSuffixes[aggType.ID()] = o.AggregationSumSqSuffix()
 		case policy.Stdev:
-			o.suffixes[aggType.ID()] = o.AggregationStdevSuffix()
+			o.defaultSuffixes[aggType.ID()] = o.AggregationStdevSuffix()
 		default:
 			q, ok := aggType.Quantile()
 			if ok {
-				o.suffixes[aggType.ID()] = o.timerQuantileSuffixFn(q)
+				o.defaultSuffixes[aggType.ID()] = o.timerQuantileSuffixFn(q)
 			}
 		}
 	}
 }
 
+func (o *options) computeCounterSuffix() {
```

Comment:
`computeCounterSuffixes` to be consistent? Also the three `computeXXXSuffixes` functions can probably be refactored into a common one `func computeSuffixes(suffixes *[][]byte, suffixOverride map[policy.AggregationType][]byte)`?